src/nvim/log.h

```diff
@@ -67,6 +67,30 @@
        __VA_ARGS__)
 #  endif
 
+# if __linux__
+#  include <execinfo.h>
+#  define LOG_CALLSTACK(prefix) \
+  do { \
+    void *trace[100]; \
+    int trace_size = backtrace(trace, 100); \
+    \
+    char exe[1024]; \
+    ssize_t elen = readlink("/proc/self/exe", exe, sizeof(exe) - 1); \
+    exe[elen] = 0; \
+    \
+    for (int i = 1; i < trace_size; i++) { \
+      char buf[256]; \
```

Comment:
What’s the point of allocating 1024 bytes for `exe` if you are trying to fit it into buffer with only 256 bytes?